src/mapnode.cpp

```diff
@@ -551,7 +551,7 @@ u8 MapNode::getMaxLevel(const NodeDefManager *nodemgr) const
 {
 	const ContentFeatures &f = nodemgr->get(*this);
 	// todo: after update in all games leave only if (f.param_type_2 ==
-	if( f.liquid_type == LIQUID_FLOWING || f.param_type_2 == CPT2_FLOWINGLIQUID)
+	if( f.liquid_type == LIQUID_FLOWING || f.param_type_2 == CPT2_FLOWINGLIQUID || f.param_type_2 == CPT2_COLORED_FLOWINGLIQUID)
```

Comment:
I feel like one or two of the conditions here ought to be redundant: `liquid_type == LIQUID_FLOWING` should already imply a paramtype2 of `flowingliquid` or `colorflowingliquid`, and those types only really make sense in conjunction with the liquid type.